plugins/registry.js

```diff
@@ -174,25 +174,23 @@ export default class PluginRegistry {
         return id;
     }
 
-    // Add a "call button"" next to the attach file button. If there are more than one button registered by any
+    // Add a "call button" to the channel header. If there are more than one button registered by any
     // plugin, a dropdown menu is created to contain all the call plugin buttons.
     // Accepts the following:
-    // - icon - React element to use as the button's icon
-    // - action - a function called when the button is clicked, passed the channel and channel member as arguments
-    // - dropdown_text - string or React element shown for the dropdown button description
-    // - tooltip_text - string shown for tooltip appear on hover
+    // - button - a React element to use as the main button to be displayed in case of a single registration.
```

Comment:
@hanzei Do we want to pass things here as React elements (both for the button and the dropdown)? This is good since gives more liberty to the plugin developer, but also may cause some problems because of that freedom. We already had some problems with the alignment of the dropdown in the channel header because we were using components styled one way or another.

Reply:
I like how it is implemented with the dedicated css classes implemented by the webapp. :+1:

We certainly could have gone that route with the App Bar as well, though it's not set in stone what will be done with the components in the far future so we wanted to be as opinionated as possible on the core webapp side.